lib/yotpo/core/response_parser.rb

```diff
@@ -4,7 +4,7 @@
 module Yotpo
   class ResponseParser < Faraday::Response::Middleware
     def parse(body)
-      unless body.empty?
+      unless body.to_s.empty?
```

Comment:
@threadhead let's use "if body.present?" instead of going though "to_s" and this confusing "unless"